testing/perl-crypt-openssl-aes/APKBUILD

```
@@ -0,0 +1,39 @@
+# Automatically generated by apkbuild-cpan, template 1
+# Maintainer: Sergey Safarov <s.safarov@gmail.com>
+pkgname=perl-crypt-openssl-aes
+_realname=Crypt-OpenSSL-AES
+pkgver=0.02
+pkgrel=0
+pkgdesc="Perl module Crypt-OpenSSL-AES"
+url="http://search.cpan.org/~ttar/Crypt-OpenSSL-AES-0.02/"
+arch="all"
+license="Artistic-1.0 GPL+"
+cpandepends=""
+cpanmakedepends=""
+depends="$cpandepends"
+makedepends="perl-dev openssl-dev $cpanmakedepends"
+install=
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/T/TT/TTAR/$_realname-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_realname-$pkgver
```

Its better to use builddir rather than _buildir because that is the automatic variable name. For more information see `man APKBUILD`.
```suggestion
builddir="$srcdir"/$_realname-$pkgver
```